### DISCUSSION TITLE
插件自身支持设置代理

### DISCUSSION THREAD

**Discussion by gufeng0** [YiiGuxing/TranslationPlugin#2055] [Ideas]
请问可以让插件自己支持设置代理吗，而不是依赖idea 的代理设置，毕竟日常开发还是需要内网的

**Comment by YiiGuxing** [comment]
呃。。。这个完全没有必要，代理的方式有很多种，可以走idea 的代理，也可以走系统的代理，而且代理也有规则，配置好了，一般使用场景能可以使用的。

**Comment by PickRecalled** [comment]
工作中我们有很多的VPN场景，比如公司的代理，科学上网的代理等，所以我觉得添加一个可以配置代理的选项是一个不错的想法，因为你可以指定某个具体的代理IP+端口

**Reply by YiiGuxing** [reply]
😂主要是代理工具本身就有这些功能的，插件再实现一个。。。插件的负担就变大了啊，我抗议！我要减负！

**Reply by meilonme** [reply]
走系统代理是个不错的选择, 但是插件目前是不走系统代理的;
我现在是用的 IDE 自带的代理, 比较无语的是 IDE 自带的代理不能设置"白名单", 只能设置"黑名单", 所以还要配置一堆不走代理的域名和IP😂
